packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+     * change. The listener will receive object with the following structure:
+     * {{
+     * jid: jid,//the JID of the participant that changed his display name
+     * displayname: displayName //the new display name
+     * }}
+     * participantJoined - receives event notifications about new participant.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * participantLeft - receives event notifications about participant the that
+     * left the room.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * @param event the name of the event
+     * @param listener the listener
+     */
+    JitsiMeetExternalAPI.prototype.addEventListener
+        = function (event, listener) {
+
+        var message = {type: "event", action: "add", events: [event]};
+        this.eventHandlers[event] = listener;
+        this.sendMessage(message);
+    };
+
+    /**
+     * Removes event listener.
+     * @param event the name of the event.
+     */
+    JitsiMeetExternalAPI.prototype.removeEventListener
+        = function (event) {
+        if(!this.eventHandlers[event])
+        {
+            console.error("The event " + event + " is not registered.");
+            return;
+        }
+        var message = {type: "event", action: "remove", events: [event]};
+        delete this.eventHandlers[event];
+        this.sendMessage(message);
+    };
+
+    /**
+     * Removes event listeners.
+     * @param events array with the names of the events.
+     */
+    JitsiMeetExternalAPI.prototype.removeEventListeners
+        = function (events) {
+        var eventsArray = [];
+        for(var i = 0; i < events.length; i++)
+        {
+            var event = events[i];
+            if(!this.eventHandlers[event])
+            {
+                console.error("The event " + event + " is not registered.");
+                continue;
+            }
+            delete this.eventHandlers[event];
+            eventsArray.push(event);
+        }
+
+        if(eventsArray.length > 0)
+        {
+            this.sendMessage(
+                {type: "event", action: "remove", events: eventsArray});
+        }
+
+    };
+
+    /**
+     * Processes message events sent from Jitsi Meet
+     * @param event the event
+     */
+    JitsiMeetExternalAPI.prototype.processMessage = function(event) {
+        var message;
+        try {
+            message = JSON.parse(event.data);
+        } catch (e) {}
+
+        if(!message.type) {
+            console.error("Message without type is received.");
+            return;
+        }
+        switch (message.type) {
+            case "system":
+                if(message.loaded) {
+                    this.onFrameLoaded();
+                }
+                break;
+            case "event":
+                if(message.action != "result" ||
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2736174701/issues/source?bid=3199634&fileBranchId=3342079#l311)